src/uct/ib/base/ib_md.c

```diff
@@ -29,6 +29,31 @@
 #include <sys/resource.h>
 #include <float.h>
 
+#if HAVE_TL_DC
+#include <uct/ib/dc/dc_mlx5.h>
+#endif
+
+#if HAVE_TL_RC
+#include <uct/ib/rc/verbs/rc_verbs.h>
+#ifdef HAVE_MLX5_HW
```

Comment:
can this be also `#if` for consistency?

Reply:
done

Reply:
still has `if` for  `HAVE_TL_DC`. BTW, what is the preferred way `if` or `ifdef`? I thought it is the former

Reply:
> still has `if` for `HAVE_TL_DC`. BTW, what is the preferred way `if` or `ifdef`? I thought it is the former

missed this - was not included into staged changes. fixed
about former:
```
#define QQQ 0

#ifdef QQQ
/* true */
#endif

#if QQQ
/* false */
#endif

#undef QQQ

#ifdef QQQ
/* false */
#endif 

#if QQQ
/* compilation warning */
#endif